mgmtfn/k8splugin/kubeClient.go

```diff
@@ -284,9 +284,10 @@ func (c *APIClient) WatchServices(respCh chan SvcWatchResp) {
 				continue
 			}
 
-			//if wss.Object.ObjectMeta.Namespace != "default" {
-			//	continue
-			//}
+			if strings.Compare(wss.Object.ObjectMeta.Namespace, "kube-system") == 0 {
```

Comment:
If you're checking for the result to be 0, there's no difference vs just using `==` here

From the docs: `The result will be 0 if a==b`